tests/test_constant_visitor.py

```diff
@@ -22,66 +22,74 @@
 
 
 import unittest
-import sys
-
-
-sys.path.append("../../")
-
 
 from src.operators.boolean_theory import *
 from src.operators.integer_theory import *
 from src.operators.real_theory import *
+from src.operators.string_theory import *
 from src.visitors.constant_visitor import ConstantVisitor
 
 
 class TestConstantVisitor(unittest.TestCase):
     def test_boolean_visitor_easy(self):
+        visitor = ConstantVisitor()
         tree = BooleanVariable('x')
-        constants = tree.accept(ConstantVisitor())
-        self.assertEqual(constants, {})
+        self.assertEqual({}, tree.accept(visitor))
         tree = BooleanConstant('c0')
-        constants = tree.accept(ConstantVisitor())
-        self.assertEqual(constants, {'c0': 'Bool'})
+        self.assertEqual({'c0': 'Bool'}, tree.accept(visitor))
         tree = BooleanEquality(BooleanConstant('c0'), BooleanConstant('c1'))
-        constants = tree.accept(ConstantVisitor())
-        self.assertEqual(constants, {'c0': 'Bool', 'c1': 'Bool'})
+        self.assertEqual({'c0': 'Bool', 'c1': 'Bool'}, tree.accept(visitor))
 
     def test_integer_visitor_easy(self):
+        visitor = ConstantVisitor()
         tree = IntegerVariable('x')
-        constants = tree.accept(ConstantVisitor())
-        self.assertEqual(constants, {})
+        self.assertEqual({}, tree.accept(visitor))
         tree = IntegerConstant('c0')
-        constants = tree.accept(ConstantVisitor())
-        self.assertEqual(constants, {'c0': 'Int'})
+        self.assertEqual({'c0': 'Int'}, tree.accept(visitor))
         tree = IntegerEquality(IntegerConstant('c0'), IntegerConstant('c1'))
-        constants = tree.accept(ConstantVisitor())
-        self.assertEqual(constants, {'c0': 'Int', 'c1': 'Int'})
+        self.assertEqual({'c0': 'Int', 'c1': 'Int'}, tree.accept(visitor))
 
     def test_real_visitor_easy(self):
-        tree = RealVariable('c0')
```

Comment:
The name `c0` was chosen on purpose to make sure the `ConstantVisitor` is not confusing a variable for a constant even if the name follows the constant naming convention.

Reply:
Ok, but why only do it for Reals? I'll change it for all of them.

Reply:
Thanks!